cfme/infrastructure/virtual_machines.py

```diff
@@ -302,12 +302,16 @@ class InfraVmReconfigureView(BaseLoggedInPage):
     cpu_total = WInput()  # read-only, TODO widgetastic
 
     disks_table = DisksTable()
+    affected_vms = WTTable(VersionPick({Version.lowest(): '//*[@id="records_div"]/table',
+                                        '5.9': '//*[@id="miq-gtl-view"]/miq-data-table/div/table'})
```

Comment:
REQUIRED: I don't think you need a version pick here.  You can combine both xpaths into one.
Please check our code for xpath examples.

Reply:
Changed.